queries/formInstances.ts

```diff
@@ -31,6 +31,36 @@ export const getFormLabel = async function (formTtl: string) {
   }
 };
 
+export const getFormPrefix = async function (formTtl: string) {
+  const q = `
+    PREFIX ext: <http://mu.semte.ch/vocabularies/ext/>
+
+    SELECT DISTINCT *
+    WHERE {
+        ?s ext:prefix ?o .
+    }
+    `;
+  const store = await ttlToStore(formTtl);
+  const engine = new QueryEngine();
+  const bindingStream = await engine.queryBindings(q, {
+    sources: [store],
+  });
+
+  const defaultPrefix = 'http://data.lblod.info/form-data/instances';
+
+  const bindings = await bindingStream.toArray();
+  if (bindings.length) {
```

Comment:
nitpick, but thought i'd share as it's a nice pattern in general. I'm a big fan of the 'return early' pattern: https://medium.com/swlh/return-early-pattern-3d18a41bba8

This keeps your code on the 'happy path', minimizing the number of constraints you have to keep in mind while reading. i.e. in this case you would go:
```
if (bindings.length === 0) {
  return defaultPrefix;
}
// now we know we're on the happy path again!
const binding = bindings[0].get('o');
if(!binding || binding.length < 1) {
  throw new Error('someone did something really bad here, better fail hard');
}
// yay happy path again
return binding.value;
```